### PR TITLE
13047 set up feis route edit components and list on projects page

### PR DESCRIPTION
**Pull request by dhochbaum-dcp**
# Big Picture Summary

### CODE REVIEW

File: client/app/components/packages/feis/edit.hbs [hunk 1]
```diff
@@ -0,0 +1,25 @@
+<div class="grid-x grid-margin-x">
+  <div class="cell large-8">
+
+    <section class="form-section">
+      <h1 class="header-large">
+        Environmental Impact Statement (EIS)
```
Comment (godfreyyeung) [inline client/app/components/packages/feis/edit.hbs:6]: Can you add "Final" here?

Comment (godfreyyeung) [reply client/app/components/packages/feis/edit.hbs:6]: This is an improvement to how the wireframe has it

**Review by allthesignals — COMMENTED** [review]
Thanks, @dhochbaum-dcp! Can you take a look at:

 1. Lint fails in the checks (if you check the bottom of the PR, the `ember test` section fails). Try to run `yarn run lint` inside the client folder.
 2. Rebase your branch with develop to address the merge conflicts. Don't hesitate to reach out to the conflicting commit's author if you're unsure how to resolve something!

Best,
Matt

**Comment by allthesignals** [comment]
@CeciliaCY why close?

**Comment by CeciliaCY** [comment]
@allthesignals, it's easier to redo this task from scratch, so I created a new branch and new pull request.

**Comment by allthesignals** [comment]
@CeciliaCY oh okay. So you're working on it now. Was confused bc it's a different PR author.

**Comment by CeciliaCY** [comment]
@allthesignals , yes, it used to be @dhochbaum-dcp 's task and has been transferred to me.

**Pull request closed without merging**